mappings/net/minecraft/client/render/LightmapTextureManager.mapping

```diff
@@ -1,4 +1,18 @@
 CLASS net/minecraft/class_765 net/minecraft/client/render/LightmapTextureManager
+	COMMENT Represents the lightmap texture manager.
```

Comment:
```suggestion
	COMMENT The lightmap texture manager maintains a texture containing the RGBA overlay for each of the 16&times;16 sky and block light combinations.
```
The general idea is to summarize what the manager actually does. Feel free to write in your own words.

Reply:
I don't know what to write instead tbh, so I'll just include that suggestion directly.